tests/plugins/basicstyles/basicstyles.js

```diff
@@ -7,6 +7,8 @@ bender.editor = {
 	}
 };
 
+var isNotGeckoOrIe = !CKEDITOR.env.gecko && !CKEDITOR.env.ie;
```

Comment:
That's interesting. It looks like it was some cache because when I rerun these tests without `isNotGeckoOrIe` in `getData` it works so, I'll remove it.